roles/ceph-common/tasks/installs/install_on_debian.yml

```diff
@@ -27,3 +27,9 @@
   when:
     - (ceph_origin == 'repository' or ceph_origin == 'distro')
     - ceph_repository == 'rhcs'
+
+- name: include install_debian_jemalloc.yml
+  include: install_debian_jemalloc.yml
+  when:
+    - (ceph_origin == 'repository' or ceph_origin == 'distro')
+    - ceph_jemalloc_enabled | default(False)
```

Comment:
you can remove `| default(False)` and add `ceph_jemalloc_enabled` to roles/ceph-defaults/default/main.yml then run `generate_group_vars_sample.sh`.

Thanks.